contributors/devel/e2e-tests.md

```diff
@@ -115,7 +114,7 @@ GINKGO_PARALLEL=y go run hack/e2e.go -- -v --test --test_args="--ginkgo.skip=\[S
 # You can also specify an alternative provider, such as 'aws'
 #
 # e.g.:
-KUBERNETES_PROVIDER=aws go run hack/e2e.go -- -v --build --up --test --down
+go run hack/e2e.go -- -v --build --up --test --down --provider=aws
```

Comment:
Is there a reason putting the flag behind? Will this work: `go run hack/e2e.go -- --provider=aws -v --build --up --test --down`?